x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -73,6 +73,7 @@ export const GET_PROCESSES_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/running_procs`
 export const KILL_PROCESS_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/kill_process`;
 export const SUSPEND_PROCESS_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/suspend_process`;
 export const GET_FILE_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/get_file`;
+export const GET_EXECUTE_ROUTE = `${BASE_ENDPOINT_ACTION_ROUTE}/execute_command`;
```

Comment:
Suggestion: name it consistent with how other commands are named - no "command" - just  "execute"

Reply:
4f5272c33dc3ea0c4cdc29f0c518c1d02cd8da9b